guide/designing-products/open-design.md

```diff
@@ -51,68 +52,105 @@ Illustration sources
 
 ## Values
 
-In open design, we:
+Open-source software development has existed for decades. Public review and debate have helped it create better products and services. Open design shares similar values, though it is not as known or practiced.
+
+### Integrity
+
+- Make moral and responsible choices
+- Build with a principled foundation
+
+### Collaboration
+
+- Collaborate without competing
+- Approach problems with an open-mind
+- Share transparent work and open processes
+- Make it easy for others to build on our work and ideas
+- Be considerate while giving and receiving feedback
 
-- collaborate without competing
-- share our process and work in a transparent way
-- make it easy for others to build on our work and ideas
-- aim for inclusivity
-- approach problems with an open-mind
-- embrace multidisciplinary feedback
-- rely on our peers for quality assurance and validation
-- fulfill the open design group’s goals through action
+### Inclusion
 
+- Aim for inclusiveness, usability, and accessibility
+- Design features for the broadest range of people, locations, backgrounds, technological knowledge, financial literacy...
+- Embrace multidisciplinary feedback
 
-We value:
+### Real world solutions
 
-- moral and responsible choices
-- products that solve real problems and address real needs
-- usability and accessibility for the broadest possible range of people, locations, and backgrounds
-- highly functional aesthetic elements that influence both how people feel and think
+- Help design products that solve real problems and address real needs
+- Rely on our peers for quality assurance and validation
+- Fulfill the open design group's goals through actions
+- Create functional elements that have a positive influence on how people feel and think
 
+## Discover the Bitcoin Design Community
 
-## Getting started
+**Our focus is on exploring best practices to improve Bitcoin product design.**
 
-If you are new to open design for Bitcoin, which many are, it can be tricky to find a place to start. Not only is the tech complex, but there are few educational resources available. The [contribute page]({{ '/contribute' | relative_url }}) is a good starting point. These tips can also help you find your footing:
+- Join the conversation on [Slack]({{site.slack_invite_url}}). Navigate toward the channels that interest you. Dive into topics and conversations that draw you in.
+- Identify your strengths and goals. Figure out how they may fit within the community's vision. A good place to start is listening to a [Community Call](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Community+Call) or reading our [newsletter](https://bitcoindesign.substack.com/).  
```

Comment:
How about following on Twitter, and also linking the YouTube channel? There's a big collection of videos that may give people a good idea of how the calls pan out. Even if they can't join due to time zones, work, family, etc, they can still get a sense from the videos.

Reply:
Done in https://github.com/BitcoinDesign/Guide/pull/588#discussion_r766540497